pyanaconda/installation.py

```diff
@@ -327,6 +325,23 @@ def run_pre_install():
 
         payload.requirements.add_groups(payload.language_groups(), reason="language groups")
         payload.requirements.add_packages(payload.langpacks(), reason="langpacks", strong=False)
+
+        # add package requirements from modules
+        # - iterate over all modules we know have valid package requirements
+        # - add any requirements found to the payload requirement tracking
+        modules_with_package_requirements = [SECURITY]
+        for module in modules_with_package_requirements:
+            module_proxy = module.get_proxy()
+            module_requirements = Requirement.from_structure_list(module_proxy.CollectRequirements())
+            for requirement in module_requirements:
+                # check requirement type and add a payload requirement appropriately
+                if requirement.type == "package":
+                    payload.requirements.add_packages([requirement.name], reason=requirement.reason)
+                elif requirement.type == "group":
+                    payload.requirements.add_groups([requirement.name], reason=requirement.reason)
```

Comment:
I recommend to implement a new method `add_requirements(requirements)` in the class `PayloadRequirements` and move this code there.

Reply:
Good point! Added. :)